test/functional/ex_cmds/write_spec.lua

```diff
@@ -82,8 +88,11 @@ describe(':write', function()
     command('let $HOME=""')
     eq(funcs.fnamemodify('.', ':p:h'), funcs.fnamemodify('.', ':p:h:~'))
     -- Message from check_overwrite
-    eq(('\nE17: "'..funcs.fnamemodify('.', ':p:h')..'" is a directory'),
-       redir_exec('write .'))
+    -- FIXME: 'E13: File exists (add ! to override)' in Windows
```

Comment:
Can this actually be fixed? Does gVim behave the same?

Reply:
`:write .` outputs the same error on gVim so it's likely expected behaviour on Windows.